app/views/citationsNotFound.html

```diff
@@ -22,13 +22,14 @@
     This site is for St. Louis County tickets.  Both the unincorporated areas and the municipalities that are part of the county.
   </p>
   <div>
-    <p>
-      Currently the following municipalities are supported by this site:
-    </p>
-    <p ng-repeat="municipality in ctrl.supportedMunicipalities">
-      <span ng-if="!$first">, </span>
-      {{municipality.name}}
-    </p>
+    <div class="commaListDiv">
+      <span>Currently the following municipalities are supported by this site:</span>
+      <br>
+      <div class="supportedMunicipalitiesDiv" ng-repeat="municipality in ctrl.supportedMunicipalities">
```

Comment:
Instead of making this inline-block and non-standard for the div element, why don't you just wrap a div element around your ng-repeat and have a span tag be your ng-repeat? Then you won't need any css change